aiofreepybox/api/netshare.py

```diff
@@ -0,0 +1,59 @@
+class Netshare:
+
+    def __init__(self, access):
+        self._access = access
+
+    server_type = [
+        'powerbook',
+        'powermac',
+        'macmini',
+        'imac',
+        'macbook',
+        'macbookpro',
+        'macbookair',
+        'macpro',
+        'appletv',
+        'airport',
+        'xserve'
+    ]
+
+    afp_configuration_schema = {
+        'enabled': False,
+        'guestAllow': False,
+        'loginName': '',
+        'loginPassword': '',
+        'serverType': server_type[0]
+    }
+
+    samba_configuration_schema = {
+        'fileShareEnabled': False,
+        'logonEnabled': False,
+        'logonPassword': '',
+        'logonUser': '',
+        'printShareEnabled': True,
+        'workgroup': 'workgroup'
```

Comment:
These should be in snake_case, not in camelCase.

Reply:
Missed it, fix incoming, thanks.